realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -68,6 +68,9 @@
 
 @RealmClass
 public abstract class RealmObject implements RealmModel, ManagableObject {
+    static final String MSG_NULL_OBJECT = "'model' is null.";
```

Comment:
This constant is used in test to check if the thrown exception is expected or not.

https://github.com/realm/realm-java/pull/4778#discussion_r121644858